Cargo.toml

```diff
@@ -114,7 +114,8 @@ libp2p-websocket = { version = "0.43.0", path = "transports/websocket" }
 libp2p-websocket-websys = { version = "0.3.0", path = "transports/websocket-websys" }
 libp2p-webtransport-websys = { version = "0.2.0", path = "transports/webtransport-websys" }
 libp2p-yamux = { version = "0.45.0", path = "muxers/yamux" }
-multiaddr = "0.18.0"
+# multiaddr = "0.18.0"
+multiaddr = { git = "https://github.com/multiformats/rust-multiaddr", rev = "d0e9ec219645ce8bd8ae7926e4b7316a67c35f84" }
```

Comment:
Release pull request: https://github.com/multiformats/rust-multiaddr/pull/103

Reply:
`multiaddr` `v0.18.1` is released. @stormshield-frb may I ask you to do a last update to the pull request to the new release?

Reply:
Sure. It's done :wink: